pkg/rpcclient/actor/actor.go

```diff
@@ -180,6 +182,12 @@ func (a *Actor) GetVersion() result.Version {
 	return *a.version
 }
 
+// GetMillisecondsPerBlock is a convenience method that returns the network's
+// milliseconds per block value.
+func (a *Actor) GetMillisecondsPerBlock() time.Duration {
```

Comment:
Why do you need it as an exported method?

Reply:
We actually don't need it, fixed.